config/formats.js

```diff
@@ -541,7 +541,15 @@ let Formats = [
 		},
 		onModifyPriority: function (priority, pokemon, target, move) {
 			// @ts-ignore
-			if (move.category !== 'Status' && pokemon && pokemon.forte) return priority + pokemon.forte.priority;
+			if (move.category !== 'Status' && pokemon && pokemon.forte) {
+				let ability = pokemon.getAbility();
+				// @ts-ignore
+				if (ability.id === 'galewings' && pokemon.forte.type === 'Flying' && pokemon.hp === pokemon.maxhp) return priority + (move.type === 'Flying' ? -1 : 1);
+				// @ts-ignore
+				if (ability.id === 'triage' && pokemon.forte.flags['heal']) return priority + (move.flags['heal'] ? -3 : 3);
```

Comment:
```suggestion
				if (ability.id === 'triage' && pokemon.forte.flags['heal']) return priority + (move.flags['heal'] ? 0 : 3);
```

either learn to math, or just test your code :p